omnigibson/robots/fetch.py

```diff
@@ -387,6 +406,40 @@ def disabled_collision_pairs(self):
         return [
             ["torso_lift_link", "shoulder_lift_link"],
             ["torso_lift_link", "torso_fixed_link"],
+            ["torso_lift_link", "estop_link"],
+            ["base_link", "laser_link"],
+            ["base_link", "torso_fixed_link"],
+            ["base_link", "l_wheel_link"],
+            ["base_link", "r_wheel_link"],
+            ["base_link", "estop_link"],
+            ["torso_lift_link", "shoulder_pan_link"],
+            ["torso_lift_link", "head_pan_link"],
+            ["head_pan_link", "head_tilt_link"],
+            ["shoulder_pan_link", "shoulder_lift_link"],
+            ["shoulder_lift_link", "upperarm_roll_link"],
+            ["upperarm_roll_link", "elbow_flex_link"],
+            ["elbow_flex_link", "forearm_roll_link"],
+            ["forearm_roll_link", "wrist_flex_link"],
+            ["wrist_flex_link", "wrist_roll_link"],
+            ["wrist_roll_link", "gripper_link"],
```

Comment:
It seems that we are explicitly disabling collision pairs between parent and child links  (e.g. wrist_roll_link is the parent and gripper_link is the child). Are we sure this is necessary?

Reply:
I'm not fully sure about that either - we are making a second pass over the whole robot copy / IK-FK / etc. mechanism really soon and I'd like to leave this as-is until then. Currently it is unclear if these are spurious collisions or real collisions that would happen on the robot too. But is for sure that the primitives do not work currently without this.